src/core/Microsoft.PowerToys.Settings.UnitTest/ViewModelTests/General.cs

```diff
@@ -50,8 +50,8 @@ public void IsElevated_ShouldUpdateRunasAdminStatusAttrs_WhenSuccessful()
             // Arrange
             GeneralViewModel viewModel = new GeneralViewModel();
             
-            string runningAsUserText = "Running as user.";
-            string runningAsAdminText = "Running as Adminstrator.";
+            string runningAsUserText = "Running as user";
```

Comment:
why aren't you grabbing the value from the resx?

Reply:
@laviusmotileng-ms :  Strings are likely to change and a unit test shouldn't need to be updated in this case.  I'd probably reference a public const variable in GeneralViewModel.

Reply:
https://github.com/microsoft/PowerToys/pull/2971/commits/60a08cff92477aa10536a9d13e1525e2c4e7df5d